src/employees.js

```diff
@@ -59,7 +61,7 @@ const formData = [
     title: "Personal Information",
     inputsList: [
       { name: "internalId", type: "string", required: true },
-      {}, // a blank space
+      { name: "active", type: "boolean", label: "Active" },
```

Comment:
Please move active button to the bottom

Reply:
![image](https://github.com/entropy-code/entropay-admin-ui/assets/46864825/ddbbf7ed-b1c4-4c79-a96f-fa1723cd0e62)